components/form/Input.js

```diff
@@ -21,7 +21,12 @@ const Input = ({ Icon, label, name, type }) => {
 };
 Input.propTypes = {
   label: PropTypes.string,
-  name: PropTypes.string,
+  name: PropTypes.string.required,
```

Comment:
I think this should be `isRequired`:

https://reactjs.org/docs/typechecking-with-proptypes.html

```suggestion
  name: PropTypes.string.isRequired,
```